libcontainer/container.go

```diff
@@ -56,9 +56,16 @@ type BaseState struct {
 	// InitProcessPid is the init process id in the parent namespace.
 	InitProcessPid int `json:"init_process_pid"`
 
-	// InitProcessStartTime is the init process start time.
+	// InitProcessStartTime is the init process start time in clock cycles since boot time.
 	InitProcessStartTime string `json:"init_process_start"`
 
+	// CreationTimeStamp is the unix timestamp for the start time of the container in UTC
+	CreationTimeStamp string `json:"creation_timestamp"`
```

Comment:
Updated to use time.Time in canonical form in memory and serialized..  now waits for put it into RFC3339Nano string format until it outputs it on the list command.